src/Console/Application.php

```diff
@@ -19,30 +18,40 @@ public function __construct()
         parent::__construct(self::NAME);
     }
 
-    /**
-     * This method override adds option to load custom config via --config in any command.
-     * And --level option as well.
-     */
     protected function getDefaultInputDefinition(): InputDefinition
     {
-        return new InputDefinition([
-            new InputArgument('command', InputArgument::REQUIRED, 'The command to execute'),
-            new InputOption(
-                '--config',
-                null,
-                InputOption::VALUE_REQUIRED,
-                'Path to config file.',
-                getcwd() . '/rector.yml'
-            ),
-            new InputOption(
-                '--level',
-                null,
-                InputOption::VALUE_REQUIRED,
-                'Finds config by shortcut name.'
-            ),
-            new InputOption('--help', '-h', InputOption::VALUE_NONE, 'Display this help message'),
-            new InputOption('--ansi', '', InputOption::VALUE_NONE, 'Force ANSI output'),
-            new InputOption('--no-ansi', '', InputOption::VALUE_NONE, 'Disable ANSI output'),
-        ]);
+        $defaultInputDefinition = parent::getDefaultInputDefinition();
+
+        $this->removeUnusedOptions($defaultInputDefinition);
+        $this->addCustomOptions($defaultInputDefinition);
+
+        return $defaultInputDefinition;
+    }
+
+    private function removeUnusedOptions(InputDefinition $inputDefinition): void
+    {
+        $options = $inputDefinition->getOptions();
+
+        unset($options['quiet'], $options['version'], $options['no-interraction']);
```

Comment:
Should be `interaction` (only one r)

Reply:
Thanks :+1: 

I wish they were constants